pkg/kp/service.go

```diff
@@ -38,3 +39,16 @@ func (c consulStore) RegisterService(manifest pods.Manifest, caPath string) erro
 
 	return c.client.Agent().ServiceRegister(podService)
 }
+
+// Go version of http status check
+func HttpStatusCheck(node string, port int) *http.Response {
+	url := fmt.Sprintf("http://%s:%d/_status", node, port)
+	resp, _ := http.Get(url)
+	return resp
+}
+
+func HttpsStatusCheck(node string, port int) *http.Response {
+	url := fmt.Sprintf("https://%s:%d/_status", node, port)
+	resp, _ := http.Get(url)
```

Comment:
I dealt with this by making the result critical if the resp is nil. In the case of resp == nil I would think we would still want the critical result. Would the best thing to do be log the error? and still write the health result.